webviz_subsurface/containers/_parameter_distribution.py

```diff
@@ -28,12 +29,15 @@ class ParameterDistribution(WebvizContainer):
 and correlation between the parameters as a correlation matrix.
 
 * `ensembles`: Which ensembles in `container_settings` to visualize.
+* `drop_constants`: Drop constant parameters (Default: true)
```

Comment:
Could drop the `(Default: true)` as that is added to the generated documentation automatically from the function signature.